presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -141,7 +141,6 @@ public static Slice stPolygon(@SqlType(StandardTypes.VARCHAR) Slice input)
     public static double stArea(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
     {
         OGCGeometry geometry = deserialize(input);
-        validateType("ST_Area", geometry, EnumSet.of(POLYGON, MULTI_POLYGON));
```

Comment:
Method's description as well as documentation in `presto-docs/src/main/sphinx/functions/geospatial.rst` need updating.

Reply:
@Description on L138 needs updating